### ISSUE TITLE
1.17.1: Bosses in Netherworlds can only spawn above Bedrock

### ISSUE THREAD

**Issue by Miryex** [kangarko/Boss#957]
Any Boss that is set to spawn in the Nether will only spawn above the Bedrock at height 128. 
Kind regards,

**Comment by Abrar879** [comment]
did u set anythin in random spawn like it and u can always limit the height

**Comment by Miryex** [comment]
Yeah I did, but that isnt the issue. If the Height limit ist set to 0-256, they only spawn above bedrock. If it is set to like 0-100 (Bedrock is at 128) they dont spawn at all :/
